.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build and Push Docker Image
+
+on:
+  pull_request:
+    branches:
+      - custom-main
+  push:
+    branches:
+      - custom-main
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build and push
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/amd64
+          push: true
+          tags: kampkelly/whisper-streaming-server:latest
```

Comment:
Please this should point to aidevices instead.

Reply:
I still can't access it anymore. Seems you need to verify the email on the account